pyEpiabm/pyEpiabm/sweep/host_progression_sweep.py

```diff
@@ -52,11 +56,50 @@ def __init__(self):
         self.delay = np.floor(self.latent_to_symptom_delay
                               / self.model_time_step)
 
+        # Infectiousness progression
+        # Instantiate parameters to be used in update infectiousness
+        infectious_profile = pe.Parameters.instance().infectiousness_prof
+        inf_prof_res = len(infectious_profile) - 1
+        infectious_period = pe.Parameters.instance().asympt_infect_period
+        # Extreme case where model time step would be too small
+        max_inf_steps = 2550
+        # k is number of time steps a person is infectious
```

Comment:
Could we use a more meaningful name than k in that case - covidsim likes the one letter variables but I think we are trying to avoid them here!
(Ditto for j, s etc)

Reply:
alright